newIDE/app/src/SceneEditor/index.js

```diff
@@ -718,8 +728,15 @@ export default class SceneEditor extends React.Component<Props, State> {
     );
   };
 
-  _startRenamingSelectedObject = () => {
-    this._onRenameObjectStart(this.state.selectedObjectsWithContext[0]);
+  _startRenamingSelectedObjectOrGroup = () => {
+    if (this.state.activeEditor === 'objects-list') {
+      this._onRenameObjectStart(this.state.selectedObjectsWithContext[0]);
+    } else if (
+      this.state.activeEditor === 'object-groups-list' &&
+      this._objectGroupsList
+    ) {
+      this._objectGroupsList.startEditingSelectedGroup();
```

Comment:
this is the thing I'm worried about. I'm unsure if this notion of "activeEditor" is the right one.

Reply:
I couldn't find something similar in react-mosaic API.